packages/AreaChart/src/chart-tooltip.scss

```diff
@@ -32,6 +33,38 @@
   --ids-chart-tooltip-circle-size: 0.8rem;
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-chart-tooltip-font-family: var(--hop-body-sm-font-family);
+  --ids-chart-tooltip-font-size: var(--hop-body-sm-font-size);
+  --ids-chart-tooltip-font-weight: var(--hop-body-sm-font-weight);
+  --ids-chart-tooltip-color: var(--hop-neutral-text);
+  --ids-chart-tooltip-background: var(--hop-neutral-surface);
+  --ids-chart-tooltip-box-shadow: var(--hop-elevation-lifted);
+  --ids-chart-tooltip-border-radius: var(--hop-shape-rounded-md);
+  --ids-chart-tooltip-padding: var(--hop-space-inset-md);
+
+  /* Date */
+  --ids-chart-tooltip-date-font-weight: var(--hop-body-sm-semibold-font-weight);
+
+  /* Score */
+  --ids-chart-tooltip-score-font-family: var(--hop-heading-lg-font-family);
+  --ids-chart-tooltip-score-font-size: var(--hop-heading-lg-font-size);
+  --ids-chart-tooltip-score-font-weight: var(--hop-heading-lg-font-weight);
+  --ids-chart-tooltip-score-line-height: var(--hop-heading-lg-line-height);
+  --ids-chart-tooltip-score-margin-top: var(--hop-space-stack-sm);
+  --ids-chart-tooltip-score-value-margin-right: var(--hop-space-inline-sm);
+  --ids-chart-tooltip-score-text-color: var(--hop-neutral-text-weak);
+  --ids-chart-tooltip-score-text-font-size: var(--hop-body-sm-font-size);
+  --ids-chart-tooltip-score-text-font-weight: var(--hop-body-sm-font-weight);
+
+  /* Circle */
+  --ids-chart-tooltip-circle-background: #BFDCA9; /* Missing semantic token */
```

Comment:
Is Design aware?

Reply:
All the ones that say missing semantic token is because they are part of these tokens https://www.figma.com/file/he7mOkVFtqk0m3dgWKLcXk/%F0%9F%90%B8-Data-visualization?type=design&node-id=40%3A1101&mode=design&t=aWERRvcA3IcZ0HRa-1
so they do exist, but not on our side.

Reply:
Should we open an issue in Hopper?